icu4j/main/classes/core/src/com/ibm/icu/text/PersonNameFormatter.java

```diff
@@ -286,7 +311,9 @@ private PersonNameFormatter(Locale locale, Length length, Usage usage, Formality
 
     /**
      * @internal For unit testing only!
+     * @deprecated This API is for technology preview only.
```

Comment:
```suggestion
     * @deprecated This API is for unit testing only.
```

Reply:
Same issue here.  This really is just for unit testing, never to be public.  I support Markus's proposed change.

Reply:
Done